index.js

```diff
@@ -1,11 +1,29 @@
-function scuberGreetingForFeet(){
+function scuberGreetingForFeet(feet){
   // Write your code here!
+  if(feet <= 400){
+    return 'This one is on me!'
+  } else if(feet >400 && feet < 2000){
```

Comment:
You can exclude `feet > 400 &&` since it's already covered by the first part of the conditional.